backend/src/main/java/pl/edu/pw/elka/etherscan/EthereumAddressValidator.java

```diff
@@ -0,0 +1,23 @@
+package pl.edu.pw.elka.etherscan;
+
+
+
+public class EthereumAddressValidator {
+
+    public boolean checkAddres(String address){
+        if(address.charAt(0) != '0' || address.charAt(1) != 'x')
```

Comment:
jak bedzie void to bedzie mozna to pokroic na mniejsze metody typu validateThatBeginsWith0x

Reply:
i starajmy sie to co jest w ifie wrzucac w {} nawet jak jest jedna linia, mozna tak uniknac glupich bledow czasami